airbyte-webapp/src/components/connection/CatalogTree/next/SyncModeSelect.module.scss

```diff
@@ -1,3 +1,6 @@
+@use "scss/variables";
+
 .pillSelect {
-  width: 100%;
+  width: variables.$width-wide-menu;
```

Comment:
100% did not work as expected?

Reply:
It did not.  Also, Nico pointed out that the SyncModeSelect should stay consistent width even when the other columns resize.